tests/AvaTax.TaxModule.Test/TaxModuleTest.cs

```diff
@@ -30,33 +30,33 @@ public class TaxModuleTest
         private const string ApplicationName = "AvaTax.TaxModule for VirtoCommerce";
         private const string ApplicationVersion = "2.x";
```

Comment:
Should it be changed to 3.0?
If these constants is not used, I think you could remove them

Reply:
changed to 3.x